examples/00_misc/05_standalone_field.py

```diff
@@ -0,0 +1,27 @@
+"""
+Standalone Field class
+----------------------
+
+The :any:`Field` class of GSTools can be used to plot arbitrary data in nD.
+
+In the following example we will produce 10000 random points in 4D with
+random values and plot them.
+"""
+import numpy as np
+import gstools as gs
+
+x1 = np.random.RandomState(19970221).rand(10000) * 100.0
+x2 = np.random.RandomState(20011012).rand(10000) * 100.0
+x3 = np.random.RandomState(20210530).rand(10000) * 100.0
+x4 = np.random.RandomState(20210531).rand(10000) * 100.0
+values = np.random.RandomState(2021).rand(10000) * 100.0
+
+###############################################################################
+# Only thing needed to instantiate the Field is the dimension.
+#
+# Afterwards we can call the instance like all other Fields
+# (:any:`SRF`, :any:`Krige` or :any:`CondSRF`), but with an additional field.
+
+plotter = gs.field.Field(dim=4)
```

Comment:
Maybe you could import gstools here instead of at the beginning of the script to emphasize that gstools is not needed to generate the field? - But this is a rather uncommon practice in Python, which could potentially confuse people...

Reply:
I would vote against importing not at the top of the script. There is also a description stating that these values are not generated with gstools.

Reply:
But with using `RNG` of gstools (as suggested in the comment above), they are again generated with gstools :joy:

Reply:
> But with using `RNG` of gstools (as suggested in the comment above), they are again generated with gstools joy

Yeah I know... but see my other comment.

Reply:
OK, so to not confuse users, I will keep it at the top.